woocommerce/woocommerce
# update Mongolia postcode to be 5 digits instead of 6 in Checkout block

Mongolia's postcode is made of 5 digits, the validation library we use indicates 6, I pushed a fix upstream but also shipping this here if that one never ships.

Closes https://github.com/woocommerce/woocommerce/issues/50251
This PR https://github.com/melwynfurtado/postcode-validator/pull/198 is merged but I still expanded this to support the XXXXX-XXXX format.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In Checkout, select Mongolia as a country.
2. Enter a zipcode that's made of 5 digits.
3. Notice that it doesn't show an error.
4. Enter a zip that's 6 digits.
5. Notice that it throws an error.
6. Enter a zipcode that's 5 digits, dash, and 4 digits.
7. Ensure there's no error.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Adjust Mongolia postcode validation to be 5 digits or 5 digits followed by 4 digits.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
